test/python/predict/test_predict.py

```diff
@@ -14,7 +14,7 @@
 def validate_metrics(config, metrics, num_items, output_dir=None):
 
     if output_dir is None:
-        metrics_file = Path(config.storage.dataset.base_directory) / Path("metrics.txt")
+        metrics_file = Path(config.storage.dataset.dataset_dir) / Path("metrics.txt")
```

Comment:
For the marius_predict module. It makes more sense to output the results to the model directory rather than the dataset directory